usb-gadget-service/udev/99-udc.rules

```diff
@@ -0,0 +1,5 @@
+#  SPDX-License-Identifier: MIT or LGPL-2.1+
+#
+# Copyright 2019 Collabora Ltd
```

Comment:
Is this on purpose (just double checking?

Reply:
I copied the file from here:
https://github.com/linux-usb-gadgets/gt

Reply:
ack